orchest-cli/orchestcli/cli.py

```diff
@@ -205,9 +205,10 @@ def cli():
     "--builder-pvc-size",
     "builder_pvc_size",
     type=click.IntRange(min=5),
-    default=50,
+    default=None,
     show_default=True,
-    help="Size of the image builder volume claim in Gi.",
+    help="Deprecated and ignored. Size of the image builder volume claim in Gi.",
+    hidden=True,
```

Comment:
```suggestion
    help="(DEPRECATED) Size of the image builder volume claim in Gi.",
```

Two points why I think this suggestion makes more sense:
* `(DEPRECATED)` is similar to how `click` implements deprecation for `Commands` and will implement it for `Options` ([see PR](https://github.com/pallets/click/pull/2271)).
* I don't think we should hide the option given that it then will no longer show up in the `--help` menu. Otherwise the message is never shown that it the option is deprecated.